inference-engine/src/vpu/myriad_plugin/myriad_executable_network.cpp

```diff
@@ -112,8 +114,7 @@ void ExecutableNetwork::Import(std::istream& strm,
         return;
     }
 
-    // TODO: better name
-    char networkName[1024] = "importedNetwork";
+    char networkName[1024] = IMPORTED_NETWORK_NAME;
```

Comment:
Why don't you use `std::string` here?

Reply:
I've just didn't changed how it was before, I guess `std::string` makes more sense to use